express-hello-world/server.ts

```diff
@@ -0,0 +1,11 @@
+import express from "express"
+
+const app = express();
+
+/* binds the middleware function to the app object with use()
+   no mount path is provided, so the funciton will execute every time the app receives a request*/
+app.use(function (req, res){
```

Comment:
I think you should get used to using an arrow function for this. It is far more common in the industry, so if you use `function` syntax your code looks dated.